.github/workflows/jira.yaml

```diff
@@ -53,25 +53,6 @@ jobs:
           echo "type=ISS" >> "$GITHUB_OUTPUT"
         fi
 
-        echo "==> Determining team labels"
```

Comment:
I am thinking we should remove this logic in favor of some kind of config file that could live in this repo. The config could have a team-to-label mapping. That way we only need to update a single config file when our team label inevitably changes in Jira again.

I am open to other ideas